### ISSUE TITLE
wdio: End browser on SIGINT 

### ISSUE THREAD

**Issue by SchwaFa** [webdriverio/webdriverio#746]
Hi,

we are running our tests using wdio with a conf.js file.

I'm currently looking for a way to close the browser when I quit the running wdio process in the terminal using CTRL+C.
Therefore, I added the following lines in the before handler:

```
process.on('SIGINT', function() {
	browser
		.end()
		.then(function() {
			process.exit(2);
		});
});
```
(I got the idea from [here](http://stackoverflow.com/a/14032965/1499145))

If I press CTRL+C, the browser exits, but another NodeJS process keeps still running causing 100% CPU usage.

How can I really close the browser and exit ALL WebdriverIO processes on SIGINT (CTRL+C)?

**Comment by christian-bromann** [comment]
Thanks for reporting this. I know how annoying it can be if you have to shutdown the webdriver process and its child processes stay alive and block the CPU. I guess the best solution would be to let the child processes check if the parent process exists or not. If not they should close the selenium session and end the process.

**Comment by SchwaFa** [comment]
Thanks for the fast reply. ;)

That sounds good. Looking forward to this feature! :)

**Comment by christian-bromann** [comment]
Released with `v3.2.1`

**Comment by SchwaFa** [comment]
Awesome! Thanks a lot! :)
